### ISSUE TITLE
Touch and stylus support

### ISSUE THREAD

**Issue by Prophet1111** [ephtracy/ephtracy.github.io#145]
I'm working on Surface Pro device and it would be great to rotate or zoom in/out with your fingers. More and more devices have touch screens and when you combine it with stylus that's a unbeatable productivity tool.

**Comment by Metadorius** [comment]
Agreed, even basic things like pinch to zoom already would be very handy. Own a Thinkpad X230 Tablet myself and it would be very cool to have at least rudimentary touch and pen support.
